test/routes/index-test.js

```diff
@@ -13,10 +13,12 @@ describe('Index page', () => {
         await app.close();
     });
 
-    it('app should show "Hello Node.js" message', async () => {
-        const response = await got(`http://${app.server.host}:${app.server.port}`);
+    it(`GET api/${process.env.ACTUAL_API_VERSION}/pulls/ should return 200 when passed valid repo`, async () => {
```

Comment:
It's nice that you try to write tests for the backend at this point, we shouldn't use production data source in tests. Don't forget to write a GitHub API mock server.

Reply:
I don't know about that rule. Thanks, I'll try.